WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -44,6 +44,7 @@ public class CoinJoinClient
 		public Kitchen Kitchen { get; }
 		public KeyManager Keymanager { get; }
 		private RoundStateUpdater RoundStatusUpdater { get; }
+		public bool InCriticalCoinJoinState { get; private set; }
```

Comment:
This variable can be read/written by multiple threads and should be synchronized. `volatile` should be enough here (https://stackoverflow.com/questions/3430757/what-is-the-volatile-keyword-used-for).

Reply:
ACK